Rakefile

```diff
@@ -29,6 +29,21 @@ namespace :test do
   end
 end
 
+task :deps do
```

Comment:
Would be nice to add a `desc` string to explain what this task does and also to have it appear in the `rake -T` task list